### DISCUSSION TITLE
[@types/openseadragon] missing ViewerEventName 'canvas-contextmenu'

### DISCUSSION THREAD

**Discussion by tchung1118** [DefinitelyTyped/DefinitelyTyped#57877] [Issues with a @types package] [answered]
Hi,

I noticed that "canvas-contextmenu" event is missing from `ViewerEventName` type in `@types/openseadragon`.
<img width="552" alt="스크린샷 2021-12-28 오후 10 08 04" src="https://user-images.githubusercontent.com/2008812/147569464-d95514c4-f042-4a93-846a-aea3967325a0.png">

Below is the screenshot of OpenSeadragon's documentation listing for [canvas-contextmenu event](https://openseadragon.github.io/docs/OpenSeadragon.Viewer.html#.event:canvas-contextmenu):
<img width="949" alt="스크린샷 2021-12-28 오후 10 08 27" src="https://user-images.githubusercontent.com/2008812/147569483-37208389-0ac9-488d-a3bf-8e381c4bffc0.png">

[a comment by an automated account is not shown]

**Comment by alvaromartmart** [comment, marked as answer]
Hi @tchung1118 , do you know if this should be present on v.2.4 or was it introduced in v.3.0?

**Reply by tchung1118** [reply]
Looking at their changelog, canvas-contextmenu was introduced in v3.0

**Comment by tchung1118** [comment]
I actually didn't realize OpenSeadragon v3 was released...😅 I see there is a draft PR about v3, so I'm guessing this can/will be included in that PR? Thanks for your fast reply!

**Reply by alvaromartmart** [reply]
Exactly, I'll link this there (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53719) so I don't forget about it 😉
